### PR TITLE
docs/guides/fedora/: initial Fedora 37 guides

### PR DESCRIPTION
**Pull request by zdykstra**
Work-in-progress Fedora installation guide.

Current issues:

* ~`auditd` in the booted BE isn't starting correctly, for some unknown reason~
* ~`dhclient eno1` won't start in the booted BE due to a permissions issue on `/run` - this could be normal behavior when starting it directly on Fedora.~

image

### CODE REVIEW

**Comment by zdykstra** [comment]
Completely revamped the guide to mimic how the official Workstation install process works - the magic of rsync saves the day.

File: docs/guides/fedora/_include/early-prep.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Early Setup
+-----------
+
+Switch to a root account
+
+.. code-block::
+
+  sudo su -
```
Comment (ahesford) [inline docs/guides/fedora/_include/early-prep.rst:8]:
```suggestion
  sudo -i
```

Comment (zdykstra) [reply docs/guides/fedora/_include/early-prep.rst]: done!

File: docs/guides/fedora/_include/efi-boot-method.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+.. code-block::
+
+   mount -t efivarfs efivarfs /sys/firmware/efi/efivars
+
+.. tabs::
+
+  .. group-tab:: Direct
+
+    .. code-block:: bash
```
Comment (ahesford) [inline docs/guides/fedora/_include/efi-boot-method.rst:12]:
```suggestion
```

**Pull request merged**